app/models/authentication_provider/keycloak.rb

```diff
@@ -2,7 +2,7 @@ class AuthenticationProvider::Keycloak < AuthenticationProvider
   self.authorization_scope = :iam_tools
   self.oauth_config_required = true
 
-  validates :realm, presence: true
-  validates :realm, format: { with: URI.regexp(%w(http https)), message: :invalid_url }
+  validates :site, presence: true
+  validates :site, format: { without: /\s/, message: :contains_whitespace }
```

Comment:
See https://github.com/3scale/porta/pull/3984/files#r1925282191

Also, the attribute is now referred to as `:site` everywhere, because with the old `:realm` the inline errors were not shown on some forms.
Now it's not necessary to use the URL regex, because `:site` is validated in the superclass `AuthenticationProvider`.

Reply:
In general, it was confusing to use both attribute names.
In some forms (e.g., "new" Keyclak for developer portal SSO) the input name was indeed "realm". But in others, (e.g. "new" SSO for admin portal - which is generic for two types) the input name is "site". So, when submitting a wrong URL, the inline error was shown correctly in one form, but not on the other.

I've changed the code to always use `:site` in form inputs, and in errors accordingly. But the label used for Keycloak is Realm.